File: java/URI_1070.java
package URI;
import java.util.Scanner;

public class URI_1070 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		int n = sc.nextInt();
		int cont = 0;
		
		for(int i = n; cont < 6; i++) {
			if(i % 2 != 0) {
				System.out.println(i);
				cont += 1;
			}
		}
		
		sc.close();

	}

}

File: kotlin/URI_1070.kt
import java.util.*

fun main(args: Array<String>) {
    val sc = Scanner(System.`in`)
    var n = sc.nextInt()
    var impares = 0
    while (impares < 6) {
        if (n % 2 != 0) {
            println(n)
            impares++
        }
        n++
    }
}
